.github/workflows/upload-cdn.yml

```
@@ -0,0 +1,72 @@
+name: Upload packages to CDN
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Publish packages"]
+    types:
+      - completed
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  CDN_BUCKET: gc-design-system-production-cdn
+  CDN_REGION: ca-central-1
+
+jobs:
+  upload-to-cdn:
+    runs-on: ubuntu-latest
```

Something that could be useful here is to check if the workflow trigger is `workflow_run` and it failed, avoid publishing the packages:
https://github.com/cds-snc/platform-forms-client/blob/90cc459ee90ca4012475eef27943d42a4a380d14/.github/workflows/prod-deploy.yml#L18

---

Could also add a similar `if` condition here:
```suggestion
    if: ${{ github.event_name != 'workflow_run' || github.event.workflow_run.conclusion == 'success' }}
    runs-on: ubuntu-latest
```